src/_customisation.scss

```diff
@@ -0,0 +1,22 @@
+:root {
+  /**
+    * Colors
+    * Colors are broken down into RGB and A variables to be combined for the main color
```

Comment:
This line can be removed. I was just playing with an idea but we didn't need it.